lms/templates/student_account/login_and_register.html

```diff
@@ -27,3 +27,4 @@
 <div class="section-bkg-wrapper">
     <div id="login-and-registration-container" class="login-register" />
 </div>
+<%include file="../register_cme/form.html" />
```

Comment:
oh ok, the conditional is in that file.

maybe a bit odd, but I guess that way we have to fork stuff less...